src/status_im2/subs/root.cljs

```diff
@@ -33,6 +33,7 @@
 (reg-root-key-sub :view-id :view-id)
 (reg-root-key-sub :screen-params :navigation/screen-params)
 (reg-root-key-sub :shared-element-id :shared-element-id)
+(reg-root-key-sub :close-lightbox :close-lightbox)
```

Comment:
close-lightbox naming is a little bit confusing, what's the purpose of this prop?

Reply:
On ios, when attempting to navigate back while zoomed in, the shared-element transition animation doesn't execute properly, so we need to zoom out first. This prop is used a signal for that. Renamed it to `exit-lightbox-signal`.

Reply:
I think we (team) had an opinion that we should not create global keywords anymore, without a qualification. This can help logically group related keywords. For example, name it `:lightbox/exit-signal` or something. Let's take advantage of this Clojure feature of keywords! I'm not saying we should nest, that's a discussion we had recently and it's a completely different topic.